src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -452,7 +452,7 @@ namespace BaseboardElectric {
             state.dataSize->DataZoneNumber = baseboard.ZonePtr;
             int SizingMethod = DataHVACGlobals::HeatingCapacitySizing;
             int FieldNum = 1;
-            std::string SizingString = baseboard.FieldNames(FieldNum) + " [W]";
+            std::string const SizingString = format("{} [W]", baseboard.FieldNames(FieldNum));
```

Comment:
I actually don't know which is better here.

Reply:
They are basically the same for a single `+` where the first object is a `string` object (as opposed to a literal like `" [W]"j`, but `format` is more efficient for multiple `+`'s and when the first object is a string literal. I would just get in the habit of using `format`.